sql/tables/AccountEarnedCMECredit.sql

```diff
@@ -11,3 +11,9 @@ create index AccountEarnedCMECredit_account_index
 
 create index AccountEarnedCMECredit_credit_index
 	on AccountEarnedCMECredit(credit);
+
+create index AccountEarnedCMECredit_earned_date_index
```

Comment:
This is the non-TZ version of the index. We do select `min(earned_date)` in the report updater so the index would be used there.